Math/Binary_Exponentiation/binaryExponentiation.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+long long int solve(long long int b, long long int p, long long int m) {
```

Comment:
don't you think the function name should be more descriptive, something which relates to binary exponentiation?